app/models/iterations/project_iterations.rb

```diff
@@ -47,7 +47,6 @@ def stories_between(start_at, end_at)
     def project_accepted_stories
       project
         .stories
-        .with_dependencies
```

Comment:
Where are the specs? We need to confirm that this method isn't returning all dependencies.